### PR TITLE
dev to main

### PR DESCRIPTION
**Pull request by girma336**
- I Added all requirement of the project

### CODE REVIEW

File: src/tests/countComment.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import getComment from '../module/countComment.js';
+
+describe('Test comment counter', () => {
```
Comment (TedLivist) [inline src/tests/countComment.test.js:3]: - [ ] All your tests are failing. Kindly fix them.

  didido

File: src/tests/itemCounter.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import itemConter from '../module/itemCounter.js';
+// console.log(itemConter())
```
Comment (TedLivist) [inline src/tests/itemCounter.test.js:2]: - [ ] Kindly remove commented code.

File: src/tests/itemReservation.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    expect(data.length).toEqual(1);
+  });
+  it('Search with id= 52813 and their length not equal to 5', async () => {
+    const data = await reserveCount('53049');
+    // console.log(data)
+    expect(data.length).not.toEqual(5);
+  });
+  it('Search with id= 52813 and their length equal to 3', async () => {
+    const data = await reserveCount('53050');
+    // console.log(data)
```
Comment (TedLivist) [inline src/tests/itemReservation.test.js:16]: - [ ] Kindly remove the commented console log codes.

File: README.md [hunk 4]
```diff
@@ -1 +1,126 @@
-# capston-project-js
\ No newline at end of file
+<a name="readme-top"></a>
```
Comment (TedLivist) [inline README.md:1]: - [ ] Kindly remove the `src/module/commentCounter.js` file since it is an empty file and so it is unneeded

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @girma336,

Good job so far!
✔️ Good documentation
✔️ Use of involvement API for likes and comments
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/countComment.test.js (hunk 1 above)
Comment (girma336) [reply src/tests/countComment.test.js:3]: - Hello 
- from mine they work

Comment (jerryowusu) [inline src/tests/countComment.test.js:3]: All your test are failing. Make sure that you have tests for all edge cases. Kindly think at least about the scenario in which there are 0 items on the page (counter function should return 0)

File: README.md [hunk 5]
```diff
@@ -1 +1,124 @@
...
+npm i -D gh-pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@girma336](https://github.com/girma336)
```
Comment (jerryowusu) [inline README.md:96]: [OPTIONAL] This is suppose to be a group work but I see only one author. Kindly add the name of the other author. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

wow so close


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jerryowusu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 5 above)
Comment (girma336) [reply README.md:96]: hi @jerryowusu 
- this is solo project their is not other coding partner

File: src/tests/countComment.test.js (hunk 1 above)
Comment (girma336) [reply src/tests/countComment.test.js:3]: hi @jerryowusu 
- how to fix it because it works from mine computer

File: README.md (hunk 5 above)
Comment (jerryowusu) [reply README.md:96]: I reviewed your project as a group collaborative project, so if you are claiming a solo project then you have to bring evidence of student success permitting you to do it alone to the next reviewer else your project will be invalidated because you need to do it with your current partner for this week
